carball/json_parser/player.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import math
```

Comment:
I would use numpy and np.nan. Not sure what the difference is, but would be more consistent I think. Might also more accurately reflect what's in the data frame if they're different

Reply:
^ Agreed, np.nan is what pandas uses by default.